http/vertx-web-client/src/test/java/io/quarkus/ts/http/vertx/webclient/VertxWebClientIT.java

```diff
@@ -102,7 +102,7 @@ public void getTimeoutWhenResponseItsTooSlow() {
     public void endpointShouldTrace() {
         final int pageLimit = 50;
         final String expectedOperationName = "/trace/ping";
-        await().atMost(1, TimeUnit.MINUTES).pollInterval(Duration.ofSeconds(1)).untilAsserted(() -> {
```

Comment:
because sometimes need more time (flaky test). Is something related to:
 `assertThat(operationNames, everyItem(containsString(expectedOperationName)));`

This assertion was already there, that should be investigated in another PR. From now let's leave it as it is and just increase the verification timeout